src/main/java/aeon/controller/Command.java

```diff
@@ -0,0 +1,297 @@
+                }
+                break;
+            case ("mark"):
+                try {
+                    markTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TASK_NOT_FOUND);
+                } catch (NumberFormatException e) {
+                    System.out.println(INVALID_INTEGER_MSG);
+                }
+                break;
+            case ("todo"):
+                try {
+                    addTodoTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TODO_DESC_ERROR);
+                }
+                break;
+            case ("deadline"):
+                try {
+                    addDeadlineTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(DEADLINE_FORMAT_ERR);
+                }
+                break;
+            case ("event"):
+                try {
+                    addEventTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(EVENT_FORMAT_ERR);
+                }
+                break;
+            case("delete"):
+                try {
+                    deleteTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TASK_NOT_FOUND);
+                } catch (NumberFormatException e) {
+                    System.out.println(INVALID_INTEGER_MSG);
+                }
+                break;
+            default:
+                System.out.println(INVALID_COMMAND);
+                break;
+            }
+            response = in.nextLine();
+        }
+        printGoodbyeMessage();
+    }
+
+    private static void readSavedTaskList(ArrayList<Task> list) {
+        File f = new File(FILE_PATH);
+        File d = new File(DIR_PATH);
+        if (!d.exists()) {
+                d.mkdir();
+        }
+        if (!f.exists()) {
+            try {
+                createTaskFile(f);
+            } catch (IOException e) {
+                System.out.println(CREATE_FILE_FAILED);
+            }
+        }
+        Scanner fileScanner = null;
+        try {
+            fileScanner = getScanner(f, fileScanner);
+        } catch (FileNotFoundException e) {
+            System.out.println(e.getMessage());
+        }
+        while (fileScanner.hasNext()) {
+
+            String taskInFile = fileScanner.nextLine();
+            String[] taskInFileArray = taskInFile.split(" ", 2);
+            String[] taskType = taskInFileArray[1].split(" ", 2);
+            String isDone = taskInFileArray[0];
+            switch (taskType[0]) {
+            case ("T"):
+                readFromFileTodo(list, taskType, isDone);
+                break;
+            case ("D"):
+                readFromFileDeadline(list, taskType, isDone);
+                break;
+            case ("E"):
```

Comment:
According to the Java coding standard, the case-switch statement should resemble (take note of the brackets used): 
```
switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}
```

Reply:
Sorry, I can't spot the mistake here. Do you mind elaborating further?